autorest/jsonrpc/stdstream.py

```diff
@@ -100,7 +101,7 @@ def list_inputs(self):
         write_message(request.json)
         return json.loads(read_message())["result"]
 
-    def get_value(self, key):
+    def get_value(self, key: str) -> str:
```

Comment:
return type is Any

Reply:
Yep you're right, I made a mistake there